sessions/2018/december/session-19.md

```diff
@@ -0,0 +1,34 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 19 (04/12/2018)
+
+- Testing to HTTP endpoints
+- Review past homeworks
+
+### Notes:
+
+#### Testing HTTP endpoints
+
+We cloned the 'Talenhaven' Roger's repository and from it he was explaining to me all concepts related with reach HTTP endpoint from tests.
+
+His approach was to buit a generic test to reach HTTP endpoints regardless the HTTP verb used.
+
+#### Homework revision
+
+Is recommended that the JS fronted shows results in text, this can be achieved using some library but the most simple thing to do is to return an JSON object in the API that contains a text with the calculated value and then use that string in the frontend.
```

Comment:
What do you mean by `results`? This sentence is very ambiguous just now, and from reading it I can't remember what it was about.

Reply:
I meant the API response values, I have updated it.